modules/apps/commerce/commerce-order-rule-api/bnd.bnd

```diff
@@ -1,6 +1,6 @@
 Bundle-Name: Liferay Commerce Order Rule API
 Bundle-SymbolicName: com.liferay.commerce.order.rule.api
-Bundle-Version: 7.3.1
+Bundle-Version: 8.0.0
```

Comment:
this major change I thin should be documented, please reach out to @riccardo-alberti to understand what to do